runelite-client/src/main/java/net/runelite/client/plugins/suppliestracker/ChargesBox.java

```diff
@@ -0,0 +1,134 @@
+package net.runelite.client.plugins.suppliestracker;
+
+import net.runelite.client.game.ItemManager;
+import net.runelite.client.util.QuantityFormatter;
+import static net.runelite.api.ItemID.*;
+
+class ChargesBox extends Box
+{
+	ChargesBox(
+			ItemManager itemManager,
+			String id,
+			SuppliesTrackerPlugin plugin,
+			SuppliesTrackerPanel panel,
+			ItemType type
+	)
+	{
+		super(itemManager, id, plugin, panel, type);
+	}
+
+	public String buildToolTip(int itemId, int qty)
+	{
+		StringBuilder tooltip = new StringBuilder();
+
+		switch (itemId)
+		{
+			case SCYTHE_OF_VITUR:
+				tooltip.append("<html>");
```

Comment:
`StringBuilder.append` returns the same stringbuilder instance, meaning you can chain `append` calls without adding a redundant lvt get instruction. (This all would matter way less if #2128 gets merged, because java 9 added new string appending optimazation)

This all has a really really neglible impact on performance (like none), but I guess I'm too perfectionistic to keep quiet about it

Assuming a stringbuilder called sb in lvt index 1
````
sb.append("abcd")
  .append("defg");
sb.append("hijk");

/*
above code would generate bytecode:
ALOAD 1 (load stringbuilder instance)
LDC "abcd" (load constant "abcd")
INVOKEVIRTUAL Stringbuilder.append(String str);
LDC "defg"
INVOKEVIRTUAL Stringbuilder.append(String str);
POP (removes the returned stringbuilder instance from the stack
ALOAD 1 (adds the stringbuilder instance it just removed from the stack back to it!)
LDC "HIJK"
INVOKEVIRTUAL Stringbuilder.append(String str);
*/

Reply:
removed unneeded tooltip calls